src/components/scripts/PopupWithForm.js

```diff
@@ -1,9 +1,13 @@
 import Popup from "./Popup.js";
 export default class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
-    super({ popupSelector });
+    super(popupSelector);
     this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._popupFormText = this._popupElement.querySelector(
```

Comment:
`this._popupFormText`

It's a submit button rather than a text


`_submitButton`

Reply:
* You haven't corrected something https://github.com/XDRO/se_project_aroundtheus/pull/1#discussion_r1373764702